### ISSUE TITLE
Support debugging of R markdown

### ISSUE THREAD

**Issue by int19h** [Microsoft/RTVS#1158]
Need to investigate what exactly we can do (if anything) to enable scenarios such as setting breakpoints and stepping in .Rmd files.

**Comment by MikhailArkhipov** [comment]
Yeah, I remember now. Actually we need to port and simplify VsContainedLanguageManager that actually handles all the view/buffer mappings, passes information in proper coordinates down and transforms coordinates back on return.
